exercises/rollback-with-saga/solution/workflow.go

```diff
@@ -0,0 +1,94 @@
+package pizza
+
+import (
+	"errors"
+	"time"
+
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/sdk/workflow"
+	"go.uber.org/multierr"
```

Comment:
I think in the docs we landed on not using `multierr ` since it doesn't work well in Temporal. For this sample I really don't know if you need it since if one step of the compensation fails can you really run the other compensation  steps?

Reply:
I'm OK not recommending it in the course content, but this example is taken more or less directly from https://github.com/temporalio/samples-go/blob/main/saga/workflow.go... the idea is that if a later step fails, all the compensation steps up to then need to run in reverse order.